config/ingestCalibs.py

```diff
@@ -1,6 +1,6 @@
 from lsst.obs.decam.ingestCalibs import DecamCalibsParseTask
 config.parse.retarget(DecamCalibsParseTask)
-config.parse.hdu = 1
+config.parse.hdu = 0
```

Comment:
Can we just delete this line entirely? My reading is that the default is zero anyways.

On a procedural note, it is probably better for this to be its own ticket rather than coopting a ticket already marked "Done". @mrawls if you create a new ticket and put this change (hopefully a deletion) on it, I will review.